samples/java-webmvc/generated-docs/index.html

```diff
@@ -2494,8 +2510,8 @@ <h4 id="_response_fields_12"><a class="anchor" href="#_response_fields_12"></a><
 </thead>
 <tbody>
 <tr>
-<td class="tableblock halign-left valign-top"><p class="tableblock"><code>id</code></p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock"><code>String</code></p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock"><code>+id+</code></p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock"><code>+String+</code></p></td>
```

Comment:
The added `+` seem to be independent of the changes in this PR. But as they end up in the documentation, I will have a look into it.

Reply:
Commit adding the `+` in Spring REST Docs: https://github.com/spring-projects/spring-restdocs/commit/7924d5fb016ea08d8d1eb10f7114de6d1c82eff6#diff-b3bbf7a9474c1af34348724b62d1b253

Asciidoctor manual on the topic: https://asciidoctor.org/docs/user-manual/#escaping-unconstrained-quotes
"The enclosure `+TEXT+` (text enclosed in pluses surrounded by backticks) is a special formatting combination in Asciidoctor. It means to format TEXT as monospace, but don’t interpolate formatting marks or attribute references in TEXT. It’s roughly equivalent to Markdown’s backticks. Since AsciiDoc offers more advanced formatting, the double enclosure is necessary."

Next step is to figure out why the escaping is not working here and the `+` ends up in the HTML.